From 93817714eedbceb2170d1ba0af728e0b86ca3a8b Mon Sep 17 00:00:00 2001
From: Jiawei Huang <jiawei@tigera.io>
Date: Sat, 27 Jul 2024 14:54:49 -0700
Subject: [PATCH] Move config files to /etc/calico/calico-fluent-bit

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6e1b36276..863df91e7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1043,7 +1043,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
 else()
   set(FLB_INSTALL_BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
   set(FLB_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}/${FLB_OUT_NAME}")
-  set(FLB_INSTALL_CONFDIR "${CMAKE_INSTALL_SYSCONFDIR}/${FLB_OUT_NAME}/")
+  set(FLB_INSTALL_CONFDIR "${CMAKE_INSTALL_SYSCONFDIR}/calico/${FLB_OUT_NAME}/")
   set(FLB_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_PREFIX}/include")
 endif()
 
-- 
2.45.2

