From face7cc97be4c087ff04aaccebea9880d469d310 Mon Sep 17 00:00:00 2001
From: Jiawei Huang <jiawei@tigera.io>
Date: Sat, 27 Jul 2024 14:57:29 -0700
Subject: [PATCH] Use Calico specific binary name and config for systemd
 service

---
 init/systemd.in | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/init/systemd.in b/init/systemd.in
index 8838bc5ce..0b2c5f1b0 100644
--- a/init/systemd.in
+++ b/init/systemd.in
@@ -1,14 +1,13 @@
 [Unit]
 Description=@FLB_PROG_NAME@
-Documentation=https://docs.fluentbit.io/manual/
+Documentation=https://docs.tigera.io/calico-enterprise/latest/about/
 Requires=network.target
 After=network.target
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/@FLB_OUT_NAME@
-EnvironmentFile=-/etc/default/@FLB_OUT_NAME@
-ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@FLB_OUT_NAME@ -c /@CMAKE_INSTALL_SYSCONFDIR@/@FLB_OUT_NAME@/@FLB_OUT_NAME@.conf
+EnvironmentFile=-/etc/calico/@FLB_OUT_NAME@/@FLB_OUT_NAME@.env
+ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@FLB_OUT_NAME@ -c @CMAKE_INSTALL_SYSCONFDIR@/calico/@FLB_OUT_NAME@/@FLB_OUT_NAME@.conf
 Restart=always
 
 [Install]
-- 
2.45.2

