--- fluentd.rb	2022-07-25 11:48:35.792000000 -0700
+++ tigera-fluentd.rb	2022-07-23 17:54:46.003766443 -0700
@@ -21,6 +21,26 @@
 require 'fluent/env'
 require 'fluent/version'
 
+#########################################################################################
+# This file is exactly the same as the original fluentd.rb, except for this code block  #
+#                             - START TIGERA CUSTOM CODE -                              #
+#########################################################################################
+require 'openssl'
+require 'fluent/tls'
+
+# Make OpenSSL run in FIPS mode, disabling certain cryptographic functions.
+OpenSSL.fips_mode = true
+puts('OpenSSL::OPENSSL_FIPS (FIPS Capable): %s' % OpenSSL::OPENSSL_FIPS)
+puts('OpenSSL::OPENSSL_LIBRARY_VERSION: %s' % OpenSSL::OPENSSL_LIBRARY_VERSION)
+puts('OpenSSL::OPENSSL_VERSION_NUMBER (base16): %s' % OpenSSL::OPENSSL_VERSION_NUMBER)
+puts('OpenSSL::VERSION (gem version): %s' % OpenSSL::VERSION)
+puts('OpenSSL.fips_mode: %s' % OpenSSL.fips_mode)
+puts('Fluent::TLS::CIPHERS_DEFAULT: %s' % Fluent::TLS::CIPHERS_DEFAULT)
+
+#########################################################################################
+#                             - END TIGERA CUSTOM CODE -                                #
+#########################################################################################
+
 $fluentdargv = Marshal.load(Marshal.dump(ARGV))
 
 op = OptionParser.new
